docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Designing Octopus HA on AWS
+description: Information on configuring Octopus High Availability hosted in AWS.
+position: 30
+---
+
+This section will walk through the different options and considerations for the components required when setting up Octopus High Availability in [AWS](https://aws.amazon.com/).
+
+:::hint
+If you are setting up Octopus High Availability on Azure or on-premises please see the following guides:
+- [Azure](/docs/administration/high-availability/design/octopus-for-high-availability-on-azure.md)
+- [On-Premises](/docs/administration/high-availability/design/octopus-for-high-availability-on-premises.md)
+:::
+
+## Setting up AWS for Octopus: High Availability 
+
+For the sake of simplicity, this guide assumes that all of the servers used for your Octopus High Availability instance are hosted in AWS.
+
+:::warning
+If you are choosing [IaaS](https://en.wikipedia.org/wiki/Infrastructure_as_a_service) on AWS then the [On-Premises](/docs/administration/high-availability/design/octopus-for-high-availability-on-premises.md) guide might be a better approach for you as you may have your Domain Controllers, SQL Database Server and load balancers in the cloud.
+:::
+
+### Compute
+
+To install Octopus nodes you will need at least to machines running Windows Server 2016 +. There’s only one choice when building virtual machines in AWS, and that’s [EC2 Instances](https://aws.amazon.com/ec2/instance-types/). There are a number of different instance types to choose from. When selecting the size of the instance, We generally find sticking with the General purpose size is the best option. Choosing the rights specs for the instance depends on how many nodes you plan to use.
```

Comment:
Minor typo here in the first sentence: `... you will need at least to machines` - two :)